booksapp.yml

```diff
@@ -25,10 +25,10 @@ spec:
       dnsPolicy: ClusterFirst
       containers:
       - name: service
-        image: buoyantio/booksapp:v0.0.1
+        image: buoyantio/booksapp:v0.0.2
```

Comment:
a bit outside the scope of this PR, but mind adding a few commands in the README.md to build these docker images?